packages/components/src/components/status-badge/status-badge.scss

```diff
@@ -29,7 +29,7 @@
         &--migrated_without_position,
         &--under_maintenance,
         &--unavailable {
-            border: 1.5px solid var(--status-warning);
+            border: 1.5px solid var(--c);
```

Comment:
what is c here?
also this effects all badges @Dhruv-deriv

Reply:
typo error